### PR TITLE
Updates to tng_dolores.py

### PR DESCRIPTION
**Pull request by isabelmontoyaa**
Updates on the header keywords for the tng/dolores spectrograph.
Changes on the 'saturation' and 'nonlinear' parameters of the detector.
Added the config_specific_par() function to set the wavelength
calibration lamps and arx_file.

Added the wavlength calibration file for tng/dolores LR-B grism made
with pypeit_identify.

### CODE REVIEW

**Review by rcooke-ast — APPROVED** [review]
Looks good to me!

File: pypeit/spectrographs/tng_dolores.py [hunk 1]
```diff
@@ -57,8 +57,8 @@ def get_detector_par(self, det, hdu=None):
             ysize           = 1.,
             platescale      = 0.252,
             darkcurr        = 0.0,
-            saturation      = 65535.,
-            nonlinear       = 0.76,
+            saturation      = 65500.,
+            nonlinear       = 0.99,
             mincounts       = -1e10,
             numamplifiers   = 1,
             gain            = np.atleast_1d(0.97),
```
Comment (rcooke-ast) [inline pypeit/spectrographs/tng_dolores.py:61]: I can't comment on the lines below, but you will need to replace the following lines:

datasec         = np.atleast_1d('[1:2045,51:]'),
oscansec        = np.atleast_1d('[2054:,51:]'),

File: pypeit/spectrographs/tng_dolores.py [hunk 2]
```diff
@@ -84,8 +84,42 @@ def default_pypeit_par(cls):
...
+            :class:`pypeit.par.parset.ParSet`: The PypeIt paramter set
+            adjusted for configuration specific parameter values.
+        """
+        
+        par = self.default_pypeit_par() if inp_par is None else inp_par
+
+        if self.get_meta_value(scifile, 'dispname') == 'LR-B':
+            par['calibrations']['wavelengths']['reid_arxiv'] = 'tng_dolores_LR-B_arx.fits'
+            # Add CdI
+            par['calibrations']['wavelengths']['method'] = 'full_template'#'holy-grail'
```
Comment (profxj) [inline pypeit/spectrographs/tng_dolores.py]: please remove the holy-grail comment

**Review by profxj — CHANGES_REQUESTED** [review]
This is great.  I'll approve once we have a test
in the DevSuite up and running.

File: pypeit/spectrographs/tng_dolores.py [hunk 3]
```diff
@@ -57,8 +57,8 @@ def get_detector_par(self, det, hdu=None):
             ysize           = 1.,
```
Comment (rcooke-ast) [inline pypeit/spectrographs/tng_dolores.py:54]: Also, please remove the commented out code on lines 27,28,29 - thanks!

**Comment by rcooke-ast** [comment]
Added a dev suite test... which passes
image

**Comment by profxj** [comment]
DevSuite companion:   https://github.com/pypeit/PypeIt-development-suite/pull/199

**Review by profxj — APPROVED** [review]
all good

**Pull request merged**